lib/cdo.rb

```diff
@@ -123,6 +127,10 @@ def codeprojects_hostedzone_id
       hostedzone_id('codeprojects.org')
     end
 
+    def preview_codeprojects_hostedzone_id
+      hostedzone_id('preview.codeprojects.org')
+    end
```

Comment:
This function grabs the first hosted zone found with this domain name, and it's possible there could be multiple. i would prefer within-cloudformation references if reasonable.